script.js

```diff
@@ -152,6 +141,11 @@ function playGame() {
   renderQuestion();
 }
 
+function endGame() {
+  // other stuff to do when game over say saving score will be done here.
```

Comment:
as there are two methods `playGame` and `renderQuestion`. For consistency I kept two methods `endGame` and `renderGameOver`. There is no advantage right now. But in the upcoming PRs all the end game data logic will be endGame function and say there are more things we need to render we'll make separate methods for them and then call them from here.